.ci/DockerFile

```diff
@@ -1,57 +1,28 @@
-# rust nightly docker image is currently broken for this project.
-# Use a known, good nightly version. Since there are no docker images for specific nightly versions, build from
-# scratch, using the same commands as the rust docker image with
-# - specific nightly version that works (see RUST_TOOLCHAIN)
-# - Added openssl, libssl-dev and pkg-config packages to compile native-tls
-# FROM rustlang/rust:${RUST_TOOLCHAIN}
-FROM debian:stretch-slim
+# Use rust latest stable along with
+# - openssl, libssl-dev and pkg-config packages to compile native-tls
+# - cargo make and cargo2junit
+# - specific nightly toolchain (required for now as JSON format output of cargo test requires it)
```

Comment:
I tried `cargo test -- -Z unstable-options --format=json` on Rust stable and it seems to work fine. So we should be able to get rid of nightly entirely!

Reply:
awesome! I'll give it a try on CI tomorrow!

Reply:
CI is happy 😃 
Rust nightly is gone 💥

Reply:
Woohoooo! Awesome!